softwarereview_policies.Rmd

```diff
@@ -157,6 +157,128 @@ Despite our best efforts to support contributed software, errors are the respons
+    information related to vulnerable populations bear responsibility to protect
+    them from likely harms.
+
+-   ***Personally identifiable or sensitive data***: The release of personally
+    identifiable or sensitive data is potentially harmful. This includes "reasonably
+    re-identifiable" data - which a motivated indiviudal could trace back to the
+    owner or creator even if the data are anonymized. This includes both cases where identifiers
+    (e.g., name, date of birth) are available as part of data, and also if unique
+    pseudonyms/screen names are linked with full-text posts, through which one
+    can link back individuals through cross-reference with other data sets.
+    
+While the best response to ethical concerns will be context-specific, these
+general guidelines should be followed by packages where the challenges above arise:
+
+-   Packages should adhere to data source's terms of use, as expressed in
+    website Terms and Conditions, "robots.txt" files, privacy policies, and
+    other relevant restrictions, and link to them prominently in package
+    documentation. Packages should provide or document functionality to adhere
+    to such restrictions (e.g., scrape from only allowed endpoints, use
+    appropriate rate limiting in code, examples, or vignettes). Note that while Terms
+    and Conditions, Privacy Policies, etc., may not provide sufficient bounds on
+    ethical usage, they can provide an outer bound.
+
+-   A key tool in addressing the risks posed in studying vulnerable populations or
+    using personally identifiable data is ***informed consent***.  Package authors
+    should support users' acquisition of informed consent when relevant. This may
+    include providing links to data source's preferred method of acquiring consent,
+    contact information of data providers (e.g. forum moderators), documentation
+    of informed consent protocols, or getting pre-approval for general uses of a
+    package.
+   
+    Note that consent is not implicitly granted just because data are accessible.
+    Accessible data are not necessarily public, as different persons and contexts
+    have different normative expectations of privacy (see work by [Social Data
+    Lab](http://socialdatalab.net/ethics-resources)).
+
+-   Packages accessing personally identifiable information should take special
+    care to follow [security best practices][Package Development Security Best Practices]
+    (e.g., exclusive use of secure internet protocols, strong mechanisms for
+    storing credentials, etc.).
+
+-   Packages that access or handle personally identifiable or sensitive data
+    should enable, document, and demonstrate workflows for de-identification,
+    secure storage, other best practices to minimize risk of harm.
+
+As standards for data privacy and research continue to evolve, we welcome input
+from authors on considerations specific to their software and supplemental
```

Comment:
where do we welcome this input

* in their pre-submission/submission
* as well as the issue tracker of this book

Reply:
In issue below.